src/app/(app)/(home)/search-filters/categories.tsx

```
@@ -0,0 +1,23 @@
+import { CategoryDropdown } from "~/app/(app)/(home)/search-filters/category-dropdown";
+
+interface CategoriesProps {
+	data: any;
+}
+
+export const Categories = ({ data }: CategoriesProps) => {
+	return (
+		<div className="relative w-full">
+			<div className="flex flex-nowrap items-center">
+				{data.map((category: any) => (
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)